### PR TITLE
setup CI

### PR DESCRIPTION
**Pull request by kiennt175**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by kiennt175** [comment]
ready

File: .sun-ci.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - yarn
+
+- name: test:phpunit
+  stage: test
+  image: sunasteriskrnd/php-workspace:7.4
+  before_script:
+  - composer install
+  script:
+  - php vendor/bin/phpunit
+
```
Comment (huyenttm-0901) [inline .sun-ci.yml]: end of file

File: config/photo.php [hunk 2]
```diff
@@ -1,5 +0,0 @@
-<?php 
-
-return [
-    'dev1' => asset('images/dev1.png'),
-];
```
Comment (huyenttm-0901) [inline config/photo.php:5]: xóa hẳn file đi nhé

**Comment by kiennt175** [comment]
ready

**Pull request merged**